docs/02-app/01-building-your-application/07-deploying/index.mdx

```diff
@@ -11,10 +11,16 @@ Congratulations! You're here because you are ready to deploy your Next.js applic
 
 `next build` generates an optimized version of your application for production. This standard output includes:
 
-- HTML files for pages using `getStaticProps` or [Automatic Static Optimization](/docs/pages/building-your-application/rendering/automatic-static-optimization)
```

Comment:
I'd say let's just make this more generic for both:

- HTML files for static pages